xds/internal/client/client_watchers_rds.go

```diff
@@ -40,8 +40,20 @@ type HeaderMatcher struct {
 	PresentMatch *bool       `json:"presentMatch,omitempty"`
 }
 
+// Route represents route with matchers and action.
+type Route struct {
+	Path, Prefix, Regex *string
```

Comment:
The xdsResolver would use this to generate the appropriate service config for the routing balancer, right?

In that case, would it make sense for this struct to also export a `ToJSON` method?